src/openrct2/world/map_generator/Noise.h

```diff
@@ -0,0 +1,20 @@
+/*****************************************************************************
+ * Copyright (c) 2025 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+namespace OpenRCT2::World::MapGenerator
+{
+    class Noise
```

Comment:
Do we really need such interface when such abstraction is not even used in the code?